src/app/assets/stylesheets/foundation-overrides.scss

```diff
@@ -119,3 +119,21 @@ tbody tr:nth-child(even) {
     min-width: 100px;
   }
 }
+
```

Comment:
mind throwing these in PostSummary.scss?

Reply:
Yes makes more sense.